test/systemtests/network_test.go

```diff
@@ -338,6 +338,7 @@ func (s *systemtestSuite) testNetworkAddDeleteTenant(c *C, encap, fwdmode string
 			}
 		}
 
+		time.Sleep(3 * time.Second)
```

Comment:
Yes, ideally we should be doing that. Our test infra does not have a way to do this for net-partition scenarios today.